pkg/config/config.go

```diff
@@ -11,9 +11,10 @@ import (
 	"time"
 
 	"github.com/caarlos0/log"
-	"github.com/goreleaser/goreleaser/internal/yaml"
 	"github.com/goreleaser/nfpm/v2/files"
 	"github.com/invopop/jsonschema"
+
+	"github.com/goreleaser/goreleaser/internal/yaml"
 )
```

Comment:
weird formatting change...

Reply:
Yeah I wasn't sure about this - `task fmt` suggested it though...